src/modules/experts/store/expertsSlice.ts

```diff
@@ -122,14 +134,28 @@ export const expertsSlice = createSlice({
     setExpertsPage: (state, action: PayloadAction<number>) => {
       state.experts.meta.pageNumber = action.payload;
     },
-    setExpertsFilters: (state, action: PayloadAction<IFilter[]>) => {
-      state.experts.filters = action.payload;
+    setExpertsRegionsFilter: (state, action: PayloadAction<IFilter>) => {
+      if (state.experts.filters) {
+        state.experts.filters.REGIONS = action.payload;
+      }
+    },
+    setExpertsDirectionsFilter: (state, action: PayloadAction<IFilter>) => {
+      const value = action.payload;
+      if (state.experts.filters) {
+        state.experts.filters.DIRECTIONS = {
+          ...state.experts.filters.DIRECTIONS,
```

Comment:
can't see what you spread here. shouln't this reducer be the same as `setExpertsRegionsFilter`?

Reply:
It's only a draft, these reducers will be the same. In this PR only regions filter is ready.